index.js

```diff
@@ -91,16 +138,42 @@ function localizeMessageTimes(originalMessage, timeMatches, timezoneOffset) {
   return convertedMessage;
 }
 
+/**
+ * Send an error to HoneyBadger
+ * @param {Error} err | an error object
+ */
+function badgerError(err) {
+  // Extract details of the error which are not stack or message
+  const { stack, code, message, ...error_details } = err;
+
+  Honeybadger.notify(
+    { stack, code, message },
+    {
+      context: {
+        timeMatches,
+        message: {
+          ts: shortcut.message.ts,
+          text: shortcut.message.text
+        },
+        channelId: shortcut.channel.id,
+        userId: shortcut.user.id,
+        teamId: shortcut.team.id,
+        error_details
+      }
```

Comment:
This context needs to be passed in, I believe. Honestly, I think keeping Honeybadger reporting on its own is probably still smarter. Most of the work here is just creating the context object, which pretty much means we have to be in-scope.